TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/Cache/RuntimeContentCache.php

```diff
@@ -65,6 +71,31 @@ public function __construct(Runtime $runtime)
     }
 
     /**
+     * @param string $key
+     * @param string $value
+     */
+    public function addTag($key, $value)
+    {
+        $key = trim($key);
+        $value = trim($value);
+        if ($key === '' || $value === '') {
+            return;
```

Comment:
not sure if that's a good idea, it's basically ignoring incorrect usage of the method which can be difficult to understand/debug.. maybe an exception is more fitting or what's the thinking behind it?